crates/re_build_search_index/src/meili.rs

```diff
@@ -0,0 +1,94 @@
+use meilisearch_sdk::{Client, Error, ErrorCode, MeilisearchError, Task};
```

Comment:
Dod they have a REST API we could use instead of depending on their enormous crate?

It would be so nice to just use `ureq` instead

Reply:
They do have a REST API. Our usage of meilisearch here is super simple so it should be possible to get rid of the SDK altogether. I'll give it a shot

Reply:
That would be awesome! ⭐